run/django/polls/test_polls.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from django.test import Client, TestCase  # noqa: 401
```

Comment:
Is there a particular reason we need to ignore the 401 flake8 error?

Reply:
This was copy and paste from https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/appengine/flexible/django_cloudsql/polls/test_polls.py#L15. I shall remove it.